s10/str_meth.rb

```diff
@@ -0,0 +1,84 @@
+# .split
+s = 'Hola, esta es una oración. 123'
+p s.split
+p s.split("a")
+aux = s.split(" ")
+puts
+aux.each {|w| puts w}
+puts
+
+# Problem
+# Longest word
+def longestW(str)
```

Comment:
Can you explain what this function is doing?

Reply:
It receives a String. Said String is separated with the Split method and stored into an Array. An auxiliary Array is defined, and the Array resultant of the Split "method" is travelled, and inside the Block, it inserts the Length of every word in the secondary Array. Later it returns the longest word by using the Max method to find the mayor Length within the Lengths Array. That "returns" the index from where the longest word is stored.